app/models/item.rb

```diff
@@ -2,4 +2,6 @@ class Item < ApplicationRecord
   belongs_to :user
   has_many :rentals
   has_many :users, through: :rentals
+
```

Comment:
can you add this as well:

  ```
validates :condition, inclusion: { in: ["Brand new", "Like new", "Used - Excellent", "Used - Good", "Used - Fair"] }
  validates :condition, inclusion: { in: ["Tops", "Bottoms", "Dressses and Jumpsuits", "Coats and Jackets", "Footwear",
                                          "Sets", "Accessories"] }
```

Reply:
yes perfect! Only thing is, you assigned both validations to the variable condition, I'm assuming one should be "condition" and the other on"category"?

Reply:
oh sorry i was duplicating it to avoid errros in syntax and forgot to update my word